app/js/components/sideNav/sideNav.directive.js

```diff
@@ -34,8 +37,21 @@ function sideNavCtrl($scope, Utils, $state, InsightsConfig) {
         config: [
             'app.config',
             'app.config-webhook-edit'
+        ],
+        policies: [
+            'app.view-policy',
+            'app.list-policies'
         ]
     };
+
+    function checkPolicies () {
+        $scope.canSeePolicies = policyAccounts[
```

Comment:
is the plan to release to beta with limited visibility?

Reply:
yes. Rob wants to make it available to certain accounts after he demos to those customers. I went ahead and setup an array of account numbers so that we can just add to that array as we make available to specific accounts.